__tests__/ReactRelayFragmentContainer-test.tsx

```diff
@@ -10,9 +10,17 @@
 
 'use strict';
 
-const React = require('react');
-const ReactTestRenderer = require('react-test-renderer');
-import { useFragment, RelayEnvironmentProvider, useRelayEnvironment } from '../lib';
+/* eslint-disable */
```

Comment:
Why this?

Reply:
for eslint for tests it is better to open an issue and make a dedicated PR